pkg/apis/experimental/v1alpha1/types.go

```diff
@@ -580,3 +580,64 @@ type IngressBackend struct {
 	// Specifies the port of the referenced service.
 	ServicePort util.IntOrString `json:"servicePort"`
 }
+
+type NodeResource string
+
+const (
+	// Percentage of node's CPUs that is currently used.
+	CpuConsumption NodeResource = "CpuConsumption"
```

Comment:
Optional: Consider using CpuUsage. I think it's more consistent with CpuRequest.